src/components/containers/PatientsList/__tests__/PatientsList.test.js

```diff
@@ -81,11 +177,19 @@ const patientsCounts = {
     },
   },
 }
+const patientsPerPageAmount = 1;
+const offset = -1;
+const handleHeaderCellClick = () => console.log('worked handleHeaderCellClick function');
```

Comment:
If this code was necessary for debugging reasons, we need to remove it before this pull request gets merged